cms/envs/production.py

```diff
@@ -591,6 +592,13 @@ def get_env_setting(setting):
 
 add_plugins(__name__, ProjectType.CMS, SettingsType.PRODUCTION)
 
+# Determines which origins are trusted for unsafe requests eg. POST requests.
+CSRF_TRUSTED_ORIGINS = ENV_TOKENS.get('CSRF_TRUSTED_ORIGINS', [])
```

Comment:
Is this setting actually needed in cms?

Reply:
I am not sure about this also. 
They had this entry [here](https://github.com/edx/edx-internal/blob/master/edx-remote-config/prod/studio.yml#L320)

Since its working fine without this settings in `cms.production.py` so good to remove from PR also.

Reply:
There are a lot of settings that are duplicated between lms and cms unnecessarily, I think because they were originally the same file.